modules/standard/FileSystem.chpl

```diff
@@ -861,6 +869,10 @@ proc isLink(out error:syserr, name: string): bool {
   extern proc chpl_fs_is_link(ref result:c_int, name: c_string): syserr;
 
   var ret:c_int;
+  var doesExist = exists(error, name);
```

Comment:
I think this will break `isLink` for a path that is a broken symlink. If you have:

```
$ ls .
a -> notHere
```
`exists('a')` should return `false`, but `isLink('a')` should be `true`.